index.bs

```diff
@@ -4269,16 +4210,94 @@ handled on the server side and do not need support from the API specified here.
 
 # Security Considerations # {#security-considerations}
 
+This specification defines a Web API and a cryptographic peer-entity authentication protocol.
+The [=Web Authentication API=] allows Web developers (i.e., "authors") to utilize the Web Authentication protocol in their
+[=registration=] and [=authentication=] [=ceremonies=].
+The entities comprising the Web Authentication protocol endpoints are user-controlled [=authenticators=] and a [=[RP]=]'s
+computing environment hosting the [=[RP]=]'s web application.
+In this model, the user agent, together with the [=[WAC]=], comprise an intermediary between [=authenticators=] and [=[RPS]=].
+Additionally, [=authenticators=] can [=attestation|attest=] to [=[RPS]=] as to their provenance.
+
+At this time, this specification does not feature detailed security considerations. However, the [[FIDOSecRef]] document provides a security analysis which is overall applicable to this specfication.
+Also, the [[FIDOAuthnrSecReqs]] document suite defines [=authenticator=] security characteristics which are overall applicable for WebAuthn [=authenticators=].
+
+The below subsections comprise the current Web Authentication-specific security considerations.
+
+
 ## Cryptographic Challenges ## {#cryptographic-challenges}
+
 As a cryptographic protocol, Web Authentication is dependent upon randomized challenges
-to avoid replay attacks. Therefore, both {MakePublicKeyCredentialOptions/challenge}}'s 
-and {{PublicKeyCredentialRequestOptions/challenge}}'s value, MUST be randomly generated
-by the [=Relying Party=] in an environment they trust (e.g., on the server-side), and the challenge in the client's
-response must match what was generated. This should be done in a fashion that does not rely
-upon a client's behavior; e.g.: the Relying Party should store the challenge temporarily
+to avoid replay attacks. Therefore, both {{MakePublicKeyCredentialOptions/challenge}}'s
+and {{PublicKeyCredentialRequestOptions/challenge}}'s value MUST be randomly generated
```

Comment:
I don't think "randomly generated" is the right term here.  For example, it might be the encrypted server state of a session.  It's not random, but serves as a good challenge because the server state includes a random nonce.

Reply:
@leshi: pls submit issue regarding the above after this is merged? thx.